ui/src/components/SampleView/SampleControls.js

```diff
@@ -181,226 +182,219 @@ export default class SampleControls extends React.Component {
     const zoom_motor = this.props.hardwareObjects[zoom_motor_uiprop.attribute];
 
     return (
-      <div style={{ display: 'flex', position: 'absolute', width: '100%' }}>
-        <div className="sample-controlls text-center">
-          <ul className="bs-glyphicons-list">
-            <li>
```

Comment:
The `ul` element is not needed (for what it's worth, it creates too much noise for screenreader users).